src/ui/src/builder/panels/BuilderBlueprintLibraryItem.vue

```diff
@@ -81,6 +81,16 @@ async function handleInstall() {
 			components,
 		});
 
+		// Track installation (fire-and-forget, don't block UX)
+		const appId = wf.writerAppId.value;
+		if (appId) {
+			writerApi
+				.trackBlueprintInstallation(orgId, props.block.id, appId)
+				.catch(() => {
+					// Silently ignore tracking failures - doesn't affect user experience
+				});
```

Comment:
You might want to log it anyway using `useLogger`

```suggestion
				.catch(useLogger().error);
```